lms/djangoapps/instructor/enrollment.py

```diff
@@ -79,7 +79,9 @@ def get_user_email_language(user):
     Return the language most appropriate for writing emails to user. Returns
     None if the preference has not been set, or if the user does not exist.
     """
-    return UserPreference.get_preference(user, LANGUAGE_KEY)
+    # Calling UserPreference directly instead of get_user_preference because the user requesting the
+    # information is not "user" and also may not have is_staff access.
```

Comment:
Good catch @wedaly. I removed UserPreference.get_preference and didn't notice this older comment while I was doing it. We'll need to rethink how this can get implemented.